.github/workflows/llama_cpp_plugin_build_and_test.yml

```diff
@@ -0,0 +1,80 @@
+name: llama_cpp_plugin_build_and_test
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
```

Comment:
You can just use `pull_request`, w/o any `types`, it covers the `synchronize` that, in turn, covers the `opened` and `reopened` types.

Reply:
Done